drivers/hmis/app/graphql/types/hmis_schema/external_form_submission.rb

```diff
@@ -0,0 +1,33 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Types
+  class HmisSchema::ExternalFormSubmission < Types::BaseObject
+    include Types::HmisSchema::HasCustomDataElements
+
+    available_filter_options do
+      arg :status, HmisSchema::Enums::ExternalFormSubmissionStatus
+      arg :submitted_date, GraphQL::Types::ISO8601Date
+    end
+
+    field :id, ID, null: false
+    field :submitted_at, GraphQL::Types::ISO8601DateTime, null: false
+    field :spam_score, Float, null: true
```

Comment:
I noticed that sometimes spam is a score (float), and other times it's a boolean, but I think (unless I've missed it) that we still need to add the translation between the two. This raises a set of ui-related questions,
- do we want to expose the float spam score to the user in the UI -- I assume yes?
- do we want that spam score float field to be settable by the user, or should they separately be able to check a box yes/no for whether they think it's spam, which I guess would then set spam score to 0 or 100?
- relatedly, but on a broader topic, the [mock](https://www.figma.com/file/F9EHR2IZYuMAlOJ9aOG2IL/OP-%2F-HMIS-%2F-Phase-2-(2024)?type=design&node-id=1883-14891&mode=design&t=s8now9TLznGWadbU-0) shows a trash button that would enable to delete a submission entirely. What is the difference in use case between marking as spam vs. deleting entirely? I understand these will be different functionally but I'm wondering how we hope users will use them. Maybe in the future we want to build a 'delete all spam' button or similar?

Reply:
Ok. All sounds good! I opened a PR [here](https://github.com/greenriver/hmis-warehouse/pull/4099) against this branch that incorporates some of this.